### PR TITLE
[Remove Stream/Optional usage] LPS-175220 module  layout-taglib

### PR DESCRIPTION
**Pull request by JanisZhang**
https://issues.liferay.com/browse/LPS-175220

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:echo

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/display/context/RenderCollectionLayoutStructureItemDisplayContext.java [hunk 1]
```diff
@@ -413,12 +412,15 @@ private Object _getContextObject() {
...
 
 		defaultLayoutListRetrieverContext.setConfiguration(_getConfiguration());
-		defaultLayoutListRetrieverContext.setContextObject(
-			Optional.ofNullable(
-				_getContextObject()
-			).orElse(
-				_httpServletRequest.getAttribute(InfoDisplayWebKeys.INFO_ITEM)
-			));
+
+		Object object = _getContextObject();
```
Comment (tinatian) [inline modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/display/context/RenderCollectionLayoutStructureItemDisplayContext.java:416]: @JanisZhang Please move null checking to _getContextObject() as this is the only usage for _getContextObject()

**Comment by tinatian** [comment]
@JanisZhang Please see my comment and resend.

Thanks.
Tina

**Pull request closed without merging**